src/Aspire.Hosting.Docker/DockerComposePublisherLoggerExtensions.cs

```diff
@@ -25,7 +25,7 @@ internal static partial class DockerComposePublisherLoggerExtensions
     [LoggerMessage(LogLevel.Information, "No resources found in the model.")]
     internal static partial void EmptyModel(this ILogger logger);
 
-    [LoggerMessage(LogLevel.Information, "Successfully generated Compose output in '{OutputPath}'")]
+    [LoggerMessage(LogLevel.Debug, "Successfully generated Compose output in '{OutputPath}'")]
```

Comment:
Without this, will people know where to look for the output?

Reply:
Yep, there was some duplication here because we emit the output in the completion message for the ReportingStep. I stepped down to debug since we surface Info logs by default now.